package.json

```diff
@@ -206,13 +203,14 @@
     "storybook": "SENTRY_UI_HOT_RELOAD='' start-storybook -p 9001 -c .storybook",
     "storybook-build": "build-storybook -c .storybook -o docs-ui/.storybook-out --quiet",
     "webpack-profile": "NO_TS_FORK=1 yarn -s webpack --profile --json > stats.json",
-    "build-derefed-docs": "node api-docs/index.js api-docs/openapi.json",
+    "install-api-docs": "cd api-docs && yarn install",
+    "build-derefed-docs": "yarn install-api-docs && node api-docs/index.js api-docs/openapi.json",
     "watch-api-docs": "sane 'yarn build-derefed-docs' api-docs",
     "build-css": "NODE_ENV=production yarn webpack --config=config/webpack.css.config.js",
     "build-acceptance": "yarn build-production",
     "build-production": "NODE_ENV=production yarn webpack --mode production --display errors-only",
     "build": "yarn build-production --output-path=public",
-    "validate-api-examples": "openapi-examples-validator ./api-docs/openapi.json --no-additional-properties"
+    "validate-api-examples": "yarn install-api-docs && cd api-docs && yarn openapi-examples-validator ./openapi.json --no-additional-properties"
```

Comment:
I don't really want to deal with `yarn` workspaces right now, but since there's only a small number of deps, I think this is fine.